docs/docs/references/slack-integration.mdx

```diff
@@ -0,0 +1,126 @@
+import SlackAI from './assets/slack-ai-analyst.png';
+import SlackAlert from './assets/slack-alert.png';
+import SlackSettings from './assets/slack-settings.png';
+import SlackChartUnfurl from './assets/slack-chart-unfurl.png';
+import SlackDashboardUnfurl from './assets/slack-dashboard-unfurl.png';
+import SlackScheduledChart from './assets/slack-scheduled-chart.png';
+import SlackScheduledDashboard from './assets/slack-scheduled-dashboard.png';
+
+---
+sidebar_label: Slack
```

Comment:
This hasn't rendered correctly:
![CleanShot 2024-08-27 at 16 34 30](https://github.com/user-attachments/assets/d2a0fffe-59de-49b4-b013-20c378dbe4e4)

Reply:
weird - i'll just remove the metadata and use the same format as all other pages since this doesn't seem to work correctly